Makefile

```
@@ -1,7 +1,15 @@
-.PHONY: test
+.PHONY: help test
+
+MAKEFLAGS += --no-print-directory
+
+UV := $(shell command -v uvx 2>/dev/null)
+
+ifeq ($(strip $(UV)),)
+$(error uvx is required to run template tests. Install uv from https://docs.astral.sh/uv/getting-started/installation/)
```

checkmake (0.3.2)

[warning] Target "$(error uvx is required to run template tests. Install uv from https" should be declared PHONY.

(phonydeclared)